dice-engine/README.md

```diff
@@ -60,21 +68,21 @@ _Parse dice expression_
 
 _Dice roll_
 
-- `rand(n)`
-- `roll(n, number_of_faces)`
-  - Returns a dice-roll map after calling rand(number_of_faces) n times 
+- `rand [n]`
+- `roll [n, number_of_faces]`
+    - Returns a dice-roll map after calling `rand [number_of_faces]` n times
 
 **Pure functions -**
 
 _Set operations_
 
-- `keep(dice-roll & n)`
+- `keep [dice-roll n]`
     - Returns a new dice-roll map with `:numeric_value` updated \
       and discarded dice moved from `:set` to `:discarded_dice`
-- `drop(dice-roll & n)`
+- `drop [dice-roll n]`
     - Returns a new dice-roll map with `:numeric_value` updated \
       and discarded dice moved from `:set` to `:discarded_dice`
-- `reroll(dice-roll & n)`
+- `reroll [dice-roll n]`
     - Returns a new dice-roll map if none of the rerolled dice match `n`,\
       otherwise recurses until none match. Discarded dice are \
       moved from `:set` to `:discarded_dice`
```

Comment:
Since we have removed `discarded_dice`, should these operations still assume that `discarded_dice` will be there?

Reply:
They shouldn't. I've updated the documentation of these functions.